libpod/volume.go

```diff
@@ -249,6 +249,17 @@ func (v *Volume) IsDangling() (bool, error) {
 	return len(ctrs) == 0, nil
 }
 
+var rootlessVolume = map[string]bool{
+	"":      true,
+	"local": true,
+	"bind":  true,
```

Comment:
tmpfs should also go here

Reply:
Added

Reply:
Why do we validate this at all? Rootless users could also use fuse based file systems. Can we just skip the check and handle errors from the mount command?

Reply:
That is what I am trying to figure out.